src/View/Widgets/SearchResults.vala

```diff
@@ -93,7 +93,7 @@ namespace Files.View.Chrome {
             }
         }
 
-        private new Gtk.Widget parent;
+        // private new Gtk.Widget parent;
```

Comment:
You shouldn't make changes which are unrelated to the issue in hand - however tempting it may be!  You can always submit a separate "clean up the code" PR which makes no substantive changes but improves the formatting.  Its best to keep the size of diff as small as possible.